crates/dbsp/src/trace/spine_async/mod.rs

```diff
@@ -395,6 +396,7 @@ where
             let start = Instant::now();
             let mut state = self.no_backpressure.wait(state).unwrap();
             state.merge_stats.backpressure_wait += start.elapsed();
+            counter!(COMPACTION_STALL_TIME).increment(start.elapsed().as_secs());
```

Comment:
Do we really want to measure stall time in integer seconds? This will account only for very long stalls.